### PR TITLE
fix dates

### PR DESCRIPTION
**Pull request by n-h-diaz**
fix bug with dates in epa imports

### CODE REVIEW

File: scripts/epa/airdata/EPA_AQI.tmcf [hunk 1]
```diff
@@ -4,7 +4,7 @@ typeOf: dcs:StatVarObservation
 variableMeasured: dcs:AirQualityIndex_AirPollutant
 observationDate: C:EPA_AQI->Date
 observationAbout: C:EPA_AQI->Place
-observationPeriod: dcs:"P1D"
+observationPeriod: dcs:P1D
```
Comment (pradh) [inline scripts/epa/airdata/EPA_AQI.tmcf:7]: it is just "P1D" (rangeIncludes of observationPeriod is Text - https://datacommons.org/browser/observationPeriod)

Comment (n-h-diaz) [reply scripts/epa/airdata/EPA_AQI.tmcf]: thanks for the explanation - this makes sense :) updated

**Comment by n-h-diaz** [comment]
thanks for the review!

**Comment by n-h-diaz** [comment]
merge

**Pull request merged**